checker/tests/nullness/java8/lambda/Issue572.java

```diff
@@ -13,6 +12,7 @@
   }
 
   public <A, B> A konst(@NonNull A a, @Nullable B b) {
+    //:: error: (argument.type.incompatible)
```

Comment:
Can we also add a passing test case?
Instead of having A and B used as type parameters in two methods, I would use unique names.
Do we even need method type parameters for konst?